runtime/src/lib.rs

```diff
@@ -723,6 +726,50 @@ impl pallet_tx_pause::Config for Runtime {
 	type WeightInfo = pallet_tx_pause::weights::SubstrateWeight<Runtime>;
 }
 
+parameter_types! {
+	pub const ProposalBond: Permill = Permill::from_percent(5);
+	pub const ProposalBondMinimum: Balance = 0;
+	pub const ProposalBondMaximum: Option<Balance> = Option::None;
+	pub const SpendPeriod: BlockNumber = 24 * DAYS;
+	pub const Burn: Permill = Permill::from_percent(1);
```

Comment:
need to set burn to 0, or have some alternative burn location, for the same reason that we need to track 100% of the relay chain token that we recieve

Reply:
done https://github.com/polkadot-dropit/node/pull/11/commits/9fc847d48a18cff810fa8f7229fb73c5cbba9e5d